CSC631-MultiplayerGameDev-Group-Project/TeamFourProject
# Feature UI lobby menu

# About
Implementation of lobby menu scene

## Testing this PR
1. Go to folder `mandy-ui`
2. Activate LobbyMenuScene
3. Observe that there is a `back` button, `join` button, and a placeholder text called 'Players current in Lobby'
